build/package/Dockerfile.typescript-toolset

```diff
@@ -0,0 +1,19 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN microdnf -y install nodejs && microdnf clean all

```

Comment:
BTW: there is a lot more in the quickstarter TS image ... e.g. to run a headless browser. It would be great to have this in here as well.

Also, I believe we should fix the nodes version.